docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:lts-alpine AS build
+WORKDIR /pack
+COPY --chown=1000 . .
```

Comment:
I had done this as a precaution. Here's my reasoning:

There are 2 users included in the `node:lts-alpine` container image: 
1. The default `root` user, which has full admin rights/permissions
2. A `node`  user (with UID=1000 and GID=1000) who has more limited permissions, but can execute node.

By default, the COPY command would add everything with `root` ownership, which means the `node` user might not be able to access everything. 

Since this step is done only to _pack_ the library, it might not be required... I'll take a closer look.

Reply:
Ok. so it doesn't seem to have an effect, so I'll remove it.